client/src/store/auth.js

```diff
@@ -0,0 +1,92 @@
+import Cookies from 'js-cookie';
+
+const SET_USER = 'authentication/SET_USER';
+const REMOVE_USER = 'authentication/REMOVE_USER';
+const SIGNUP ='SIGNUP';
+
+export const setUser = (user) => {
+  return {
+    type: SET_USER,
+    user
+  }
+
+
+}
+
+export const removeUser = () => {
+  return {
+    type: REMOVE_USER
+  }
+}
+
+const newUser = (user) => ({
+  type: SIGNUP,
+  user
+})
```

Comment:
since action for `newUser` does the exact same thing as `setUser` you don't need `newUser` action and can use `setUser` wherever you use `newUser`